internal/test/e2e/run.go

```diff
@@ -205,7 +205,7 @@ func (e *E2ESession) runTests(regex string) (testCommandResult *testCommandResul
 	command := "GOVERSION=go1.16.6 gotestsum --junitfile=junit-testing.xml --raw-command --format=standard-verbose --hide-summary=all --ignore-non-json-output-lines -- test2json -t -p e2e ./bin/e2e.test -test.v"
 
 	if regex != "" {
-		command = fmt.Sprintf("%s -test.run \"%s\"", command, regex)
+		command = fmt.Sprintf("%s -test.run \"%s\" -test.timeout 300m", command, regex)
```

Comment:
This sets the e2e timeout to 300 minutes for each e2e test instance

Reply:
Does this cover the duration of the individual tests or all the tests being run? If it's the timeout for all the tests, should we make it 6 hours? 5.5 is pretty close to the normal time it's been taking to run a few of the provider test durations.